src/main/java/org/apache/commons/cli/HelpFormatter.java

```diff
@@ -588,7 +605,9 @@ public String getArgName() {
      * Gets the 'descPadding'.
      *
      * @return the 'descPadding'
+     * @deprecated text formatting is now handled by {@link org.apache.commons.cli.help.TextStyle}
```

Comment:
If you import the class, you don't need a FQCN in each link tag. This would reduce the verbose nature of all these comments.